<!-- -*- mode: XML -*- -->
<launch>
	<arg name="racecar_version" default="racecar-v2" />
	
	<include file="$(find zed_wrapper)/launch/zed.launch" />
	<include file="$(find racecar)/launch/teleop.launch" />	

	<param name="push_multiplier" value="25.5" />
	<param name="steer_grad_proportion" value="1000.0" />
	<param name="speed_grad_proportion" value="-0.001" />
	<param name="momentum_mu" value="0.98" />
	<param name="update_influence" value="0.09" />
	<param name="min_speed_clamp" value="-0.7" />
	<param name="max_speed_clamp" value="1.5" />
	<param name="reverse_speed_multiplier" value="-2.3" />

	<node name="potential_field_node" pkg="racecar_63" type="potential_field.py" />
	
	<node name="blob_detection_node" pkg="racecar_63" type="color_one.py"/>
	
        <!--
	<node name="image_matching_node" pkg="racecar_63" type="image_matching.py"/>
	-->
</launch>
